hoomd/md/methods.py

```diff
@@ -152,69 +166,96 @@ class NPT(_Method):
+      tetragonal symmetry.
+    - Specifying no couplings and all degrees of freedom amounts to a fully 
+      deformable triclinic unit cell
 
-    :py:class:`NPT` uses the proper number of degrees of freedom to compute the temperature and pressure of the system in
-    both 2 and 3 dimensional systems, as long as the number of dimensions is set before the :py:class:`NPT` command
-    is specified.
 
     For the MTK equations of motion, see:
 
     * `G. J. Martyna, D. J. Tobias, M. L. Klein  1994 <http://dx.doi.org/10.1063/1.467468>`_
     * `M. E. Tuckerman et. al. 2006 <http://dx.doi.org/10.1088/0305-4470/39/19/S18>`_
     * `T. Yu et. al. 2010 <http://dx.doi.org/10.1016/j.chemphys.2010.02.014>`_
-    * Glaser et. al (2013), to be published
+    *  Glaser et. al (2013), unpublished
 
-    Both *kT* and *P* can be variant types, allowing for temperature/pressure ramps in simulation runs.
 
     :math:`\tau` is related to the Nosé mass :math:`Q` by
 
     .. math::
 
         \tau = \sqrt{\frac{Q}{g k_B T_0}}
 
-    where :math:`g` is the number of degrees of freedom, and :math:`k_B T_0` is the set point (*kT* above).
+    where :math:`g` is the number of degrees of freedom, and :math:`k_B T_0` is 
+    the set point (*kT* above).
 
-    A :py:class:`hoomd.compute.thermo` is automatically specified and associated with *group*.
 
     Examples::
-        integrator = integrate.NPT(filter=filter.All(), tau=1.0, kT=0.65, tauS = 1.2, S=2.0)
+
+        npt = hoomd.md.methods.NPT(filter=hoomd.filter.All(), tau=1.0, kT=0.65, 
+        tauS = 1.2, S=2.0)
         # orthorhombic symmetry
-        integrator = integrate.NPT(filter=filter.All(), tau=1.0, kT=0.65, tauS = 1.2, S=2.0, couple="none")
+        npt = hoomd.md.methods.NPT(filter=hoomd.filter.All(), tau=1.0, kT=0.65, 
+        tauS = 1.2, S=2.0, couple="none")
         # tetragonal symmetry
-        integrator = integrate.NPT(filter=filter.All(), tau=1.0, kT=0.65, tauS = 1.2, S=2.0, couple="xy")
+        npt = hoomd.md.methods.NPT(filter=hoomd.filter.All(), tau=1.0, kT=0.65, 
+        tauS = 1.2, S=2.0, couple="xy")
         # triclinic symmetry
-        integrator = integrate.NPT(filter=filter.All(), tau=1.0, kT=0.65, tauS = 1.2, S=2.0, couple="none", rescale_all=True)
+        npt = hoomd.md.methods.NPT(filter=hoomd.filter.All(), tau=1.0, kT=0.65, 
+        tauS = 1.2, S=2.0, couple="none", rescale_all=True)
+        integrator = hoomd.md.Integrator(dt=0.005, methods=[npt], forces=[lj])
+
+    Attributes:
+        filter (hoomd.filter._ParticleFilter): Subset of particles on which to
+            apply this method.
+
+        kT (hoomd.variant.Variant): Temperature set point for the 
+            thermostat. (in energy units).
+
+        tau (float): Coupling constant for the thermostat (in time units).
+
+        S (List[hoomd.variant.Variant]): Stress components set 
+            point for the barostat (in pressure units).
+            In Voigt notation: [Sxx, Syy, Szz, Syz, Sxz, Sxy].
```

Comment:
Mention being able to set an isotropic pressure using `npt.S = 4.`. or any arbitrary float.